src/container.cpp

```diff
@@ -85,6 +85,11 @@ Container* Container::getParentContainer()
 	return thing->getContainer();
 }
 
+std::string Container::getName() const {
```

Comment:
I think we could add bool addArticle here defaulted to false like its done in item class
so std::string Container::getName(bool addArticle /* = false*/) const {
and pass that variable to getNameDescription call

Reply:
added and tested